docs/docs/writing-your-own-enclave-host.md

```diff
@@ -118,16 +114,18 @@ public class MyEnclaveTransport implements EnclaveTransport, Closeable {
     }
 }
 ```
-
-The `input` and `output` streams are our communication channels with the host server for receiving and sending raw 
-bytes, respectively.
+This sample implements `Closeable` on the host and port parameters to allow the caller to close any underlying 
```

Comment:
I don't understand `and port parameters`, i.e. the sentence does not seem fluent to me. What do we mean?

Reply:
I'm trying to simplify this sentence in the [old text](https://docs.conclave.net/writing-your-own-enclave-host.html#implementing-the-client):

"Host and port parameters will be required, and it should implement Closeable to allow the caller to close any underlying connections."

It tries to explain the code below:

public class MyEnclaveTransport implements EnclaveTransport, Closeable {
    private final String host;
    private final int port;
    private Socket socket;
    private DataInputStream input;
    private DataOutputStream output;

    public SocketEnclaveTransport(String host, int port) {
        this.host = host;
        this.port = port;
    }

    public void start() throws IOException {
        socket = new Socket(host, port);
        input = new DataInputStream(socket.getInputStream());
        output = new DataOutputStream(socket.getOutputStream());
    }

    @Override
    public void close() throws IOException {
        if (socket != null) {
            socket.close();
        }
    }
}

Reply:
Is the following sentence clearer?

"This sample implements `Closeable` on the host and port parameters of `SocketEnclaveTransport()` method to allow the caller to close any underlying connections."